CHANGELOG.md

```diff
@@ -56,6 +56,9 @@ All notable changes for each version of this project will be documented in this
         - The `igxGrid` `onEditDone` now exposes arguments of type `IGridEditEventArgs`. The arguments expose `row` and `cell` objects where if the editing is performed on a cell, the edited `cell` and the `row` the cell belongs to are exposed. If row editing is performed, the `cell` object is null. In addition the `currentValue` and `newValue` arguments are exposed. If you assign a value to the `newValue` in your handler, then the editing will conclude with the value you've supplied.
         - The `igxGrid` `onSelection` now correctly propagates the original `event` in the `IGridCellEventArgs`.
     - Added `jsZip` as a Peer Dependency.
+- `primaryKey` attribute added to `igxGrid`
+    - `primaryKey` allows for a property name from the `data` source to be specified. If specified, `primaryKey` can be used instead of `index` to indentify grid rows from the `igxGrid.rowList`. As such, `primaryKey` can be used for selecting rows for the following `igxGrid` methods - `deleteRow`, `updateRow`, `updateCell`, `getCellByColumn`, `getRowByKey`
```

Comment:
Remove the ` ` from `data`. Explain that the `primaryKey` is better suited for uniquely identifying data members, especially when the DOM is virtualized, because indices are reused in the virtual DOM scenario.

Reply:
Thanks for pointing those out! They're addressed in the latest commit.